recipes/yamlordereddictloader/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set name = "yamlordereddictloader" %}
+{% set version = "0.4.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 7f30f0b99ea3f877f7cb340c570921fa9d639b7f69cba18be051e27f8de2080e
+
+build:
+  noarch: generic
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv "
+
+requirements:
+  host:
+    - pip
+    - python
+    - pyyaml
```

Comment:
pyyaml can be removed here afaik

Reply:
Right, should not be required for building.